strawberryfields/backends/states.py

```diff
@@ -1110,6 +1148,30 @@ def poly_quad_expectation(self, A, d=None, k=0, phi=0, **kwargs):
 
         return mean, var
 
+    def number_expectation(self, modes):
+        """
+        Calculates the expectation value of the product of the number operators of the modes.
+        Args:
```

Comment:
```suggestion
        This method computes the analytic expectation value
        :math:`\langle \hat{n}_{i_0} \hat{n}_{i_1}\dots \hat{n}_{i_{N-1}}\rangle`
        for a (sub)set of modes `[i_0, i_1, \dots, i_{N-1}]` of the system.
        
        Args:
```

Reply:
I don't like the word analytical, if it is not analytical then what is it?

Reply:
Good point, have edited the above to remove it!